src/uct/tcp/tcp_sockcm_ep.h

```diff
@@ -24,6 +24,7 @@ typedef enum uct_tcp_sockcm_ep_state {
     UCT_TCP_SOCKCM_EP_FAILED                      = UCS_BIT(13), /* ep is in error state due to an internal local error */
     UCT_TCP_SOCKCM_EP_CLIENT_GOT_REJECTED         = UCS_BIT(14), /* ep on the client side received a reject from the server
                                                                     (debug flag) */
+    UCT_TCP_SOCKCM_EP_PACK_CB_FAILED              = UCS_BIT(15)  /* the upper layer's priv_pack_cb failed */
```

Comment:
can we avoid yet another state flag?

Reply:
@yosefe yes, in case uct_cm_ep_pack_cb() fails, instead of setting a flag, we can handle the error locally there (set ep to failed and remove the async handler) and return OK from uct_tcp_sockcm_ep_pack_priv_data() so that the err_handling won't be invoked. 
imho returning UCS_OK in case of an error isn't good and handling the error right after the failure will create code duplication.
seemed to me that a flag would be simpler.
wdyt?

Reply:
i'll do the change from the comment below with keeping this flag then?

Reply:
ok